devicetypes/smartthings/ezex-temp-humidity-sensor.src/ezex-temp-humidity-sensor.groovy

```diff
@@ -16,11 +16,12 @@
 import physicalgraph.zigbee.zcl.DataType
 
 metadata {
-    definition(name: "eZEX Temp & Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn:"SmartThings", vid:"generic-humidity") {
+    definition(name: "eZEX Temp & Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn:"SmartThings", vid:"eZEX-TempHumidity_Sensor") {
```

Comment:
I think, at this point, it might make sense to either assume the default vid of `SmartThings-smartthings-eZEX_Temp_&_Humidity_Sensor and *not* declare an mnmn and vid, or to create `generic-humidity-3` as follows:

```
{
    "custom": {
        "Temperature Measurement$1": {
            "name": "Temperature Measurement",
            "range": [
                -20.0,
                100.0
            ]
        }
    },
    "dpResources": [
        "Relative Humidity Measurement",
        "Temperature Measurement$1"
    ],
    "ma": [
        "Relative Humidity Measurement"
    ],
    "ms": [
        "Relative Humidity Measurement"
    ],
    "name": "Generic Humidity Presentation Resource",
    "type": "MultiFunctionalSensor",
    "vid": "generic-humidity-3"
}
```

Reply:
I'm generating a generic-humidity-3 right now, by the way

Reply:
There is now a `generic-humidity-3`

Reply:
@dkirker u created a new ui metadata("generic-humidity-3"), and pushed it to server?

Reply:
Yes, that is correct.

Reply:
okay,  i will use it

Reply:
hm...what's more, when using "generic-humidity", the device card will not display device status, so i use "eZEX-TempHumidity_Sensor" which created by Ms Wang, Does "generic-humidity-3" can fix it?